src/pages/ReviewedCases.js

```diff
@@ -45,7 +45,7 @@ class ReviewedCases extends Component {
       }
     };
     this.getReviewedConsents = this.getReviewedConsents.bind(this);
-    this.openAccessResultRecords = this.openAccessResultRecords.bind(this);
+    this.openAccessReviewResults = this.openAccessReviewResults.bind(this);
     this.openDulResultsRecord = this.openDulResultsRecord.bind(this);
```

Comment:
These lines are legacy and unnecessary code. Both `openAccessReviewResults` and `openDulResultsRecord` are anonymous functions so they don't need to be bound to `this`. `getReviewedConsents` is NOT an anonymous function, so it DOES need to be bound to `this`. Wherever I see non-anonymous functions that are bound in the constructor like this, I generally make them anonymous and remove the bind statements - it's less boilerplate and easier to read, IMHO.